plonky3/src/stark.rs

```diff
@@ -91,9 +91,18 @@ impl<T: FieldElement> Plonky3Prover<T> {
         // get fixed columns
         let fixed = &self.fixed;
 
-        if fixed.is_empty() {
-            return;
```

Comment:
This still needs to happen in the case where we have no fixed columns and no public inputs. In that case we don't need keys.